### PR TITLE
Password Reset Rule

### PR DESCRIPTION
**Pull request by ghost**
Update Force Password Reset Rule guidance to account for changes in session invalidation

### CODE REVIEW

File: articles/rules/guides/redirect.md [hunk 1]
```diff
@@ -108,74 +108,61 @@ Suppose you would like to force users to change their passwords under specific c
...
+   *    - CLIENT_ID
+   *    - CLIENT_SECRET
+   *    - ISSUER
+   */
+
+  const url = require('url@0.10.3');
+  const req = context.request;
+
+  function mustChangePassword() {
+    // TODO: implement function
```
Comment (jeffreylees) [inline articles/rules/guides/redirect.md:128]: Quick question - was this meant to be left here

Comment (ghost) [reply articles/rules/guides/redirect.md:128]: @jeffreylees Yessir, that's meant to be here for the developer to implement the function. If it's causing confusion I can take it out - let me know :)

Comment (jeffreylees) [reply articles/rules/guides/redirect.md:128]: No, all good, just making sure you meant it as a note for them and not yourself, I write TODOs in the docs all the time but for me 🤣

**Review by jeffreylees — APPROVED** [review]

**Comment by luuuis** [comment]
@jeffreylees we still have the old guidance in https://auth0.com/docs/rules/guides/redirect#force-password-change-example. Could you investigate why?

**Comment by ghost** [comment]
Looks like there was a reegression/unintentional revert here: https://github.com/auth0/docs/commit/6c4e03b16ad9d92990fe703b5c8ee12f8a6d8d69

**Pull request merged**